src/Compilers/CSharp/Portable/Symbols/Synthesized/GeneratedNames.cs

```diff
@@ -496,6 +496,16 @@ internal static string LambdaCopyParameterName(int ordinal)
             return "<p" + StringExtensions.GetNumeral(ordinal) + ">";
         }
 
+        internal static string AnonymousDelegateParameterName(int index, int parameterCount)
+        {
+            // SPEC: parameter names arg1, ..., argn or arg if a single parameter
```

Comment:
A bit of irony, it looks like `System.Func'2` uses `arg`, but `System.Action'1` uses `obj`...
[https://learn.microsoft.com/en-us/dotnet/api/system.action-1?view=net-6.0](https://learn.microsoft.com/en-us/dotnet/api/system.action-1?view=net-6.0)

We should stick with `arg` as planned ;-)